create_ccx.py

```diff
@@ -0,0 +1,111 @@
+"""
+Script to create a CCX using the CCXCon REST APIs
+"""
+
+import argparse
+import sys
+
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+
+HTTP_201_CREATED = 201
+CCXCON_TOKEN_URL = '/o/token/'
+CCXCON_CREATE_CCX = '/api/v1/ccx/'
+
+
+def parse_auguments():
+    """
+    Parsing arguments necessary to create the CCX
+    """
+    parser = argparse.ArgumentParser(description="Create a CCX using CCXCon")
+    # Parameters for CCXCon information
+    parser.add_argument('--ccxconurl', dest='ccxconurl',
+                        help="The CCXCon URL to be used", required=True)
+    parser.add_argument('--clientid', dest='clientid',
+                        help="The user's OAUTH ID", required=True)
+    parser.add_argument('--clientsecret', dest='clientsecret',
+                        help="The user's OAUTH SECRET", required=True)
+    # Parameters for creating the CCX
+    parser.add_argument('--mastercourse', dest='mastercourse',
+                        help="The master course UUID", required=True)
+    parser.add_argument('--useremail', dest='useremail',
```

Comment:
Can we change these to use dashes for different words? Ie `--user-email` or `--client-secret`